Image-Factory/Factory.ps1

```diff
@@ -509,7 +571,18 @@ function RunTheFactory
             Copy-Item "$($ResourceDirectory)\bits" -Destination ($driveLetter + ":\") -Recurse;
             
             # Create first logon script
-            $updateCheckScriptBlock | Out-String | Out-File -FilePath "$($driveLetter):\Bits\Logon.ps1" -Width 4096;
+            if($UseStaticIP) {
+                $staticUpdateCheckScriptBlock = $updateCheckScriptBlock | Out-String
+                $staticUpdateCheckScriptBlock = $staticUpdateCheckScriptBlock.Replace('$UseStaticIP = $false', '$UseStaticIP = $true')
```

Comment:
Good point - I did it that way so if not using static IPs, the scriptblock can just be run as is.  But I can see how it would be confusing to anyone who didn't write that.

Give me a bit and I'll fix it up.